docs/mobile/exercises/multiple-choice.md

```diff
@@ -0,0 +1,19 @@
+# Multiple Choice Exercises
+
+The student has to choose one or more correct answers.
+
+## Example
```

Comment:
Before you dig into the example yaml for each exercise type, it might be useful to make a note of which fields are compulsory/optional, and what they mean. (Maybe link back to their full description in `/mobile/exercises/README.md`.)

Reply:
I have a full table for this at the top level README. Putting the relevant subset in the page for each exercise type is a good idea.